### PR TITLE
Added header section and created hw03_carnesa.py.

### PR DESCRIPTION
**Pull request by ACBerea**

### CODE REVIEW

**Comment by sheggen** [comment]
Leave the Pull Request in Github. Your code is empty though (it's the stubs file I gave you with none of your own code). Usually a Pull Request is issued after coding is complete, to signal to the repo owner (me) that it is ready for review. 

Fix your code, and commit and push. Your PR gets updated automatically with new changes. Message me again when these fixes are complete.

**Comment by ACBerea** [comment]
> Leave the Pull Request in Github. Your code is empty though (it's the stubs file I gave you with none of your own code). Usually a Pull Request is issued after coding is complete, to signal to the repo owner (me) that it is ready for review.
> 
> Fix your code, and commit and push. Your PR gets updated automatically with new changes. Message me again when these fixes are complete.

**Comment by ACBerea** [comment]
> Leave the Pull Request in Github. Your code is empty though (it's the stubs file I gave you with none of your own code). Usually a Pull Request is issued after coding is complete, to signal to the repo owner (me) that it is ready for review.
> 
> Fix your code, and commit and push. Your PR gets updated automatically with new changes. Message me again when these fixes are complete.

Apologies for taking so long. I am ready for review.

**Comment by ACBerea** [comment]
@sheggen Apologies for writing another comment. I just wanted to make sure that the other comment had been sent properly after I had accidentally closed the conversation.

File: hw03_carnesa.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+    cry.forward(99)
+    cry.left(64)
+    cry.forward(46)
+    cry.circle(25, -100)
+    cry.backward(58)
+    cry.right(13)
+    cry.forward(90)
+    cry.end_fill()
+    cry.hideturtle()
+    # ...
```
Comment (sheggen) [inline hw03_carnesa.py:211]: Remove help text from the template.

Comment (sheggen) [reply hw03_carnesa.py:211]: There are multiple of these, check for them.

Comment (ACBerea) [reply hw03_carnesa.py]: @sheggen The additional text has been removed.

**Comment by sheggen** [comment]
No need to apologize, PRs are a conversation back-and-forth!

**Pull request merged**